thenewboston_node/accounts/views/transactions.py

```diff
@@ -0,0 +1,53 @@
+from drf_spectacular.utils import OpenApiParameter, extend_schema
+from more_itertools import ilen
+from rest_framework.exceptions import ParseError
+from rest_framework.mixins import ListModelMixin
+from rest_framework.viewsets import GenericViewSet
+
+from thenewboston_node.accounts.serializers.transaction import TransactionSerializer
+from thenewboston_node.business_logic.blockchain.base import BlockchainBase
+from thenewboston_node.business_logic.models import CoinTransferSignedChangeRequest
+from thenewboston_node.core.pagination import CustomLimitOffsetPagination
+from thenewboston_node.core.utils.itertools import SliceableCountableIterable
+
+
+class TransactionViewSet(ListModelMixin, GenericViewSet):
+    serializer_class = TransactionSerializer
+    pagination_class = CustomLimitOffsetPagination
```

Comment:
Let's create another pagination class. That does not require `count`. Since for transactions it will be really slow.